* A problem with the --no-bitmap-icon argument has been fixed.
  Patch: fix-bitmap-icon-arg.diff
  Provided-by: Sven Joachim <sven_joachim@web.de>
  Date: Sun, 22 Jul 2007 17:16:09 +0200
  Added-by: Rob Browning <rlb@defaultvalue.org>
  Status: appears to be incorporated upstream

  Upstream changelog entry:

  2007-07-22  Jan Djärv  <jan.h.d@swipnet.se>

        * startup.el (command-line-x-option-alist): Use x-handle-no-bitmap-icon.

        * term/x-win.el (x-handle-no-bitmap-icon): New function.

Index: sid/lisp/startup.el
===================================================================
--- sid.orig/lisp/startup.el
+++ sid/lisp/startup.el
@@ -144,7 +144,7 @@
     ("--foreground-color" 1 x-handle-switch foreground-color)
     ("--background-color" 1 x-handle-switch background-color)
     ("--mouse-color" 1 x-handle-switch mouse-color)
-    ("--no-bitmap-icon" 0 x-handle-switch icon-type nil)
+    ("--no-bitmap-icon" 0 x-handle-no-bitmap-icon)
     ("--iconic" 0 x-handle-iconic)
     ("--xrm" 1 x-handle-xrm-switch)
     ("--cursor-color" 1 x-handle-switch cursor-color)
Index: sid/lisp/term/x-win.el
===================================================================
--- sid.orig/lisp/term/x-win.el
+++ sid/lisp/term/x-win.el
@@ -129,6 +129,9 @@
 			initial-frame-alist)
 		  x-invocation-args (cdr x-invocation-args)))))))
 
+(defun x-handle-no-bitmap-icon (switch)
+  (setq default-frame-alist (cons '(icon-type) default-frame-alist)))
+
 ;; Make -iconic apply only to the initial frame!
 (defun x-handle-iconic (switch)
   (setq initial-frame-alist
